plugins/woocommerce/includes/tracks/class-wc-site-tracking.php

```diff
@@ -74,8 +75,7 @@ public static function add_tracking_function() {
 
 				var eventName = '<?php echo esc_attr( WC_Tracks::PREFIX ); ?>' + name;
 				var eventProperties = properties || {};
```

Comment:
Part of me wants to say to update these `var`'s to `let/const`, but definitely not required 😅

Reply:
Thanks, @joelclimbsthings!  Since these are remnants from when older versions of IE were supported, I think they are safe to remove.